include/zigbee/zigbee_app_utils.rst

```diff
@@ -58,21 +58,16 @@ In general, using the default signal handler is worth considering because of the
 
 The default signal handler also serves as a good starting point for a custom signal handler implementation.
 
-.. _zarco_signal_handler_full:
-
-Complete zboss_signal_handler implementation
-============================================
+.. _zarco_signal_handler_extending:
 
-In its complete implementation, the ``zboss_signal_handler`` allows the application to control a broader set of basic functionalities, including joining, commissioning, and network formation.
+Extending zboss_signal_handler
+==============================
 
 There are cases in which the default handler is not sufficient and needs to be extended.
-For example, when the application wants to use the procedure of the initiator of finding & binding or use the production configuration feature.
-
-Extending zboss_signal_handler
-++++++++++++++++++++++++++++++
+For example, when the application wants to use the procedure of the initiator of Finding & Binding or use the production configuration feature.
 
-If you want to extend ``zboss_signal_handler`` to cover additional functionalities, write signal handler implementation for the required ZBOSS signal.
-For example for a device that initiates a F&B procedure, extend ``zboss_signal_handler`` with a case for ``ZB_BDB_SIGNAL_FINDING_AND_BINDING_INITIATOR_FINISHED``:
+If you want to extend `zboss_signal_handler()`_ to cover additional functionalities, write signal handler implementation for the required ZBOSS signal.
+For example, for a device that initiates the F&B procedure, extend ``zboss_signal_handler`` with a switch case statement for ``ZB_BDB_SIGNAL_FINDING_AND_BINDING_INITIATOR_FINISHED``:
```

Comment:
Do we want to change \`\`zboss_signal_handler\`\` to \`zboss_signal_handler()\`_ here and in other occurencies?